JustAnotherVoiceChat.Server.Wrapper/src/Elements/Client/VoiceClient.EventListener.cs

```diff
@@ -51,7 +51,11 @@ private void DetachFromStatusChangeEvents()
 
         private void OnClientConnected(TClient client)
         {
-            ExecuteOnMe(client, () => { Connected = true; });
+            ExecuteOnMe(client, () => 
+            {
+                Connected = true;
+                MuteForAll(_lastMuteForAllState);
```

Comment:
You should also lock the variable here, if you locked it there:
https://github.com/AlternateLife/JustAnotherVoiceChat-Server/pull/94/files#diff-8d74fcc9e2ea4479f61db3a3a31a1477R43